tests/testthat/test_compile_report.R

```diff
@@ -9,7 +9,7 @@ test_that("Compilation can handle multiple outputs", {
   setwd(tempdir())
   random_factory(include_examples = TRUE)
 
-  compile_report(list_reports(pattern = "foo")[1], quiet = TRUE)
+  compile_report(list_reports(pattern = "foo")[1], quiet = TRUE, other = "test")
```

Comment:
There is no `other` argument in `compile_report` (I think, can't find it added in this PR); it would be passed through `...`, but unsure what the expected behaviour is here.

Reply:
i was manually making sure that env() was capturing `...`  I'll add a check for this in the tests

Reply:
correction: this isn't adding much - could take it out or leave it...

Reply:
```r
> f <- function(...) {as.list(environment())}
> f(other = 1)
list()
> 
```

Reply:
i updated to get the dots later